### DISCUSSION TITLE
make project structure simple

### DISCUSSION THREAD

**Discussion by sahasrara62** [h2oai/wave#628] [Ideas]
#### Is your feature request related to a problem? Please describe

why integrate the go and python part together? I think it is better if both of there setup and config file are maintained in different folder respectively, easy to handle and cleaner structure

#### Describe the solution you'd like

A much more simple project folder structure where python and go component are maintained in a different folder and make the project folder structure more user friendly

**Comment by lo5** [comment]
@sahasrara62 - is there some specific problem you're running into with your IDE?

**Comment by sahasrara62** [comment]
@lo5  no problem till now, but my question is related to structuring the wave package in the base directory where a lot of go files are present, can we arrange or reorganize them in the sep go folder to run a server? as I find the whole folder can be structure in a more modular way?

**Comment by lo5** [comment]
Wave is primarily a Go project, and currently the subdirs are for language-specific drivers. We can refactor if it bothers contributors, but it's not super critical right now. Thanks for your input.

**Comment by sahasrara62** [comment]
Thanks, I think in the future when this project scale, it's better to refactor according to language specific
